lib/node/http.js

```diff
@@ -125,7 +125,9 @@ class HTTP extends Server {
         chain: {
           height: this.chain.height,
           tip: this.chain.tip.rhash(),
-          progress: this.chain.getProgress()
+          progress: this.chain.getProgress(),
+          spv: this.chain.options.spv,
```

Comment:
SPV is more a property of the node as it affects other features of the node too and not just the chain, and similar to pruned. What about here?
```js
{
  "version": "1.0.2",
  "network": "regtest",
  "chain": {
    "height": 200,
    "tip": "03c1d2ebd85058180da927dc66ad3fb862dd9e188f3de6d0b53161d538a382cc",
    "progress": 0.9975950784699014
  },
  // (...)
  "prune": false,
  "spv": false
}
```

Via this in lib/node/http.js:
```js
  // (...)
  },
  memory: this.logger.memoryUsage(),
  prune: this.node.prune,
  spv: this.node.spv,
});
```
The property should already be available for `node.spv`, and `node.pruned` would need to be, it's used multiple places so would be a good move to make.
```

Reply:
Would it be worth including the pruned depth? It's a constant network property now, but it seems like something that would be configurable.

Reply:
This seems like a good idea to have `prune` set at the node level like `spv` and reported at the root of the `getInfo` object, will update. Prune HEIGHT is available in `rpc getblockchaininfo` the prune depth is set by `keepBlocks` in `networks.js` for now. It's set to 288 blocks for mainnet and that is the standard minimum proposed by [BIP159](https://github.com/bitcoin/bips/blob/master/bip-0159.mediawiki) for `NODE_NETWORK_LIMITED` peers, which we don't actually implement yet, but will soon :-)